### ISSUE TITLE
Configuration.set should by default dump the data indented

### ISSUE THREAD

**Issue by kvanhijf** [openvstorage/framework-extensions#39]

**Comment by kvanhijf** [comment]
https://github.com/openvstorage/framework/pull/1819
https://github.com/openvstorage/alba-asdmanager/pull/295
https://github.com/openvstorage/framework-extensions/pull/40

**Comment by kvanhijf** [comment]
Default behavior in Configuration.set changed from
```
if raw is False:
    data = json.dumps(data)
```
to
```
if raw is False:
    data = json.dumps(data, indent=4)
```

**Comment by kvanhijf** [comment]
Included in release 0.2.2
